modules/azure/api_connectors/sftp_ssh/outputs.tf

```diff
@@ -0,0 +1,3 @@
+output "name" {
+  value = var.connection_name
```

Comment:
Is this name correct?
Because [here](https://github.com/recognizegroup/terraform/pull/428/files#diff-449a3c13e021e47d2619ac775b5eb8714994acc836806fcc2f994b42f3ffd27cR19) it is `"${var.connection_name}_deployment"`

Reply:
yes, I think so we have same setup also for the other api connections:
![image](https://github.com/user-attachments/assets/1b7aecf9-cf65-4664-814f-c2ee6e29d563)
![image](https://github.com/user-attachments/assets/99b80574-edf5-49d1-83f9-da79a23ee53e)

Reply:
and "_deployment" is used here 

![image](https://github.com/user-attachments/assets/ab20945e-49a1-459a-9c27-683c736192c3)